NickvisionTubeConverter.Shared/Models/Download.cs

```diff
@@ -97,49 +97,18 @@ public static async Task<bool> GetIsValidVideoUrlAsync(string url)
                         { "quiet", true },
                         { "merge_output_format", "/" }
                     };
-                    ytdlp.YoutubeDL(ytOpt).extract_info(url, download: false);
+                    Python.Runtime.PyDict videoInfo = ytdlp.YoutubeDL(ytOpt).extract_info(url, download: false);
+                    return (true, videoInfo.HasKey("title") ? (videoInfo["title"].As<string>() ?? "Video") : "Video");
```

Comment:
`As<string?>()` will make it become null if it's None in python, so it will not raise null exception and the block will return "Video" as expected.

Reply:
Yes the `??` operator means if this object is null use this value instead so in this case we want, if the string is null (None in python) return "Video".

Using `string` for `As<>` instead of `string?` will never cause it actually be null, it will just crash as @fsobolev mentioned.